samples/client/petstore/typescript-node/npm/model/models.ts

```diff
@@ -117,7 +117,7 @@ export class ObjectSerializer {
     public static deserialize(data: any, type: string) {
         // polymorphism may change the actual type.
         type = ObjectSerializer.findCorrectType(data, type);
-        if (data == undefined) {
+        if (data === undefined) {
```

Comment:
samewise, code below tries to access properties of `data`, which can still be null (it it's a nullable type as defined in OpenAPI Spec v3)